From 29020c3ccf83992621dd96e7a9d5d90d8d790181 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Thu, 29 Jun 2017 01:08:03 -0500
Subject: [PATCH] define ssize_t as int

---
 libraries/liblmdb/mdb_stat.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libraries/liblmdb/mdb_stat.c b/libraries/liblmdb/mdb_stat.c
index a7315d8..f7b0d95 100644
--- a/libraries/liblmdb/mdb_stat.c
+++ b/libraries/liblmdb/mdb_stat.c
@@ -19,6 +19,7 @@
 
 #ifdef	_WIN32
 #define	Z	"I"
+#define	ssize_t	int
 #else
 #define	Z	"z"
 #endif
-- 
2.5.1.windows.1

