From 47f4256fa9c180d34d9a5ec1b6eaf1c207f7cb94 Mon Sep 17 00:00:00 2001
From: "Viacheslav Ch." <vchychyrko@anaconda.com>
Date: Thu, 12 Feb 2026 12:49:22 +0200
Subject: [PATCH] fix aclocal exact tool pin

---
 Makefile.devel | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.devel b/Makefile.devel
index baee4ba..345a835 100644
--- a/Makefile.devel
+++ b/Makefile.devel
@@ -13,8 +13,8 @@ SHELL = /bin/sh
 MAKE = make
 AUTOCONF = autoconf
 AUTOHEADER = autoheader
-AUTOMAKE = automake-1.17
-ACLOCAL = aclocal-1.17
+AUTOMAKE = automake
+ACLOCAL = aclocal
 GPERF = gperf
 
 # Ignore the built-in default for CC. Also ignore $(CC) if it is empty.
-- 
2.50.1 (Apple Git-155)

